models/gas/polygon/gas_polygon_fees_traces.sql

```diff
@@ -13,19 +18,19 @@ WITH traces AS (
      SELECT traces.block_time
      , traces.block_number
      , traces.tx_hash
-     , MAX(traces.from) AS trace_from
+     , MAX(traces."from") AS trace_from
      , MAX(traces.to) AS trace_to
      , traces.trace
      , MAX(traces.input) AS trace_input
-     , substring(MAX(traces.input),1,10) AS trace_method
+     , bytearray_substring(MAX(traces.input),0,5) AS trace_method
```

Comment:
We may use 1,4 here for the parameters:
https://dune.com/queries/2872808?category=canonical&blockchain=polygon

```suggestion
     , bytearray_substring(MAX(traces.input),1,4) AS trace_method
```